.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -96,6 +96,21 @@ jobs:
           command: test
           args: --no-default-features
 
+  kani:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Verify with Kani
+        uses: model-checking/kani-github-action@02764e122079011adaf31e7d0630a8841d277507
```

Comment:
@danielsn what is the way to always get the latest?

Reply:
I think "always getting latest" may not be best for this setting. All other action `@version` in this file is a hard-coded version tag, which I think was intended to reduce breakage from dependency changes.

One way to implement this will be to have a rolling `@latest` tag that rolls up to the latest every release. Then `@latest` will work fine.

Reply:
a new release was done yesterday - 0.19. the commit string is quite ugly to read IMHO. If we do want to stick to a version, then maybe we use the version number. 

yes, having @latest would be a good thing for us.

Reply:
> a new release was done yesterday - 0.19. the commit string is quite ugly to read IMHO. If we do want to stick to a version, then maybe we use the version number.
I agree. I changed this to `v0.19`.

This feature seems to already be planned by DSN: https://github.com/model-checking/kani-github-action/issues/12. Re-posting the link to the action